### ISSUE TITLE
How can I download videos using yt-dlp, starting and ending at a certain point in time?

### ISSUE THREAD

**Issue by kiroffYT** [yt-dlp/yt-dlp#11124]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

I need to download a stream from YouTube. But I don't need the whole stream, just a certain part of it. For example, starting at 5:01 and ending at 16:43. How to do it?

_in Russian (На русском):_

Мне нужно скачать стрим с YouTube. Но мне нужен не весь стрим, лишь определённая его часть. К примеру, начиная с 5:01 и заканчивая 16:43. Как это сделать?

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

**Comment by grqz** [comment]
tl;dr: `--download-sections "*5:01-16:43"`
```
--download-sections REGEX       Download only chapters that match the
                                regular expression. A "*" prefix denotes
                                time-range instead of chapter. Negative
                                timestamps are calculated from the end.
                                "*from-url" can be used to download between
                                the "start_time" and "end_time" extracted
                                from the URL. Needs ffmpeg. This option can
                                be used multiple times to download multiple
                                sections, e.g. --download-sections
                                "*10:15-inf" --download-sections "intro"
```

**Comment by kiroffYT** [comment]
> tl;dr: `--download-sections "*5:01-16:43"`
> 
> ```
> --download-sections REGEX       Download only chapters that match the
>                                 regular expression. A "*" prefix denotes
>                                 time-range instead of chapter. Negative
>                                 timestamps are calculated from the end.
>                                 "*from-url" can be used to download between
>                                 the "start_time" and "end_time" extracted
>                                 from the URL. Needs ffmpeg. This option can
>                                 be used multiple times to download multiple
>                                 sections, e.g. --download-sections
>                                 "*10:15-inf" --download-sections "intro"
> ```

Thanks for the help. To be honest, I saw a similar question from another person, but not quite about it, and I was wondering whether to write "5mins 30 secs" or something else.
